include/deal.II/multigrid/multigrid.templates.h

```diff
@@ -164,32 +164,24 @@ Multigrid<VectorType>::level_v_step (const unsigned int level)
 //  std::cout<<std::endl;
 //  t[level].print(std::cout, 2,false);
 
-  // make t rhs of lower level The
-  // non-refined parts of the
-  // coarse-level defect already
-  // contain the global defect, the
-  // refined parts its restriction.
-  for (unsigned int l = level; l>minlevel; --l)
+  if (edge_out != 0)
```

Comment:
can you leave a comment here? I guess the whole following block does something like:
"compute and transfer the defect to one level coarser (and apply edge_in/out if existing)"